generator/src/main/java/com/graphicsfuzz/generator/tool/GenerateShaderFamily.java

```diff
@@ -128,6 +128,7 @@ public static void mainHelper(String[] args) throws ArgumentParserException,
     final boolean writeProbabilities = ns.getBoolean("write_probabilities");
     final boolean keepBadVariants = ns.getBoolean("keep_bad_variants");
     final boolean stopOnFail = ns.getBoolean("stop_on_fail");
+    final boolean vulkan = ns.getBoolean("vulkan");
```

Comment:
I would rename the receiving variable to `isVulkan`.

Reply:
done, also in places not mentioned in this request